scipy/sparse/tests/test_argmax.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+from numpy.testing import run_module_suite, assert_equal, assert_raises
+from scipy.sparse import bsr_matrix, coo_matrix, csr_matrix, csc_matrix
+
+
+def test_argmax():
+    D1 = np.array([
+        [-1, 5, 2, 3],
+        [0, 0, -1, -2],
+        [-1, -2, -3, -4],
+        [1, 2, 3, 4],
+        [1, 2, 0, 0],
+    ])
+
+    D2 = D1.transpose()
+
+    classes = [bsr_matrix, coo_matrix, csr_matrix, csc_matrix]
```

Comment:
When you integrate this with `test_base.py`, you can add the argmin/argmax tests to the `_TestMinMax` class and follow the existing tests as a template.